<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>200</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Корни квадратного уравнения</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-calculator"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Расчитать корни уравнения</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="factor_a">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите коэффициенты уравнения: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Результат :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result_1">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>45</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x^2 +</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="factor_b">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="factor_c">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> x +</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>= 0;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2й корень</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1й корень</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="message">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
